frontend/packages/dev-console/integration-tests/features/topology/topology-chart-area-visual.feature

```diff
@@ -172,3 +172,86 @@ Feature: Topology chart area
               And user hovers on Add to Project and clicks on From Channel
               And user clicks on Create
              Then user is able to see different applications created from Samples, From Git, Container Image, From Dockerfile, From Devfile, From Catalog, Database, Operator Backed, Helm Charts, Event Source, Channel
+
+
+        @regression, @manual
```

Comment:
Drag and drop is possible in automation, maybe you can remove the tag @manual and add @to-do

Reply:
But how do we implement it in ci

Reply:
hmm agreed, to upload the jar file from local is not possible